SortingAlgorithms/index.html

```diff
@@ -10,11 +10,11 @@
 
 	<table border="1">
 		<tr>
-			<td>1</td>
-			<td>2</td>
-			<td>3</td>
-			<td>4</td>
-			<td>5</td>
+			<td>1 Hi</td>
```

Comment:
All of these were needed because the table seemed TOO boring!

Reply:
Looks good!

Reply:
<img src="https://media0.giphy.com/media/MNmyTin5qt5LSXirxd/giphy.gif"/>